### ISSUE TITLE
[Tree Spread] Trees have no seeds anymore

### ISSUE THREAD

**Issue by GCrafter** [bcmpinc/StardewHack#104]
The mod still prevents trees from spreading and the "OnlyPreventTapped" toggle works too, but trees don't give seeds anymore at all. Only if completely chopped down. Tried it with the mod disabled and trees give seeds again if you shake them. It might be since the SDV 1.6.3 update but really not sure.
SMAPI log shows no errors or anything , can send it if needed tho.

As a side note, in the Generic Mod Config Menu the slider for the seed chance now goes from 1 to 10 instead of from 0 to 1 like in SDV 1.5.6, unsure if that is intended

**Comment by bcmpinc** [comment]
SDV 1.6 added more options for modding, including a seed percentage. Rather than overriding that I changed the config into a multiplier. Hence the 1x-10x option. With it set to 10 it basically should always have a seed. I have to look into this a bit more.

**Comment by Kevin-from-HP** [comment]
Yeah I have the same issue. I'm not entirely sure on how the rest of  the mod is behaving, but tree seeds are definitely off atm
